static/app/utils/discover/fields.tsx

```diff
@@ -964,7 +969,7 @@ export function generateFieldAsString(value: QueryFieldValue): string {
 }
 
 export function explodeField(field: Field): Column {
-  const results = explodeFieldString(field.field);
+  const results = explodeFieldString(field.field, field.alias);
```

Comment:
If you want to keep it around, I dont think we should store results to const, we can just `return explodeFieldString(...`